### PR TITLE
Completed Challenge

### PR DESCRIPTION
**Pull request by ErikAGriffin**

### CODE REVIEW

File: spec/problem_spec.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+require 'problem'
+
+describe 'Prime Problem' do
+
+  let(:primer) {Problem.new}
+
```
Comment (deniseyu) [inline spec/problem_spec.rb]: a sprinkle of rspec magic: leave out instantiation unless having distinct, multiple instances is relevant to your tests (i.e. let(:bike) { ... } and let(:broken_bike) { ... }

instead... see comment below

File: spec/problem_spec.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  it 'Knows if a number is prime' do
+
+    expect(primer.prime?(2)).to eq true
```
Comment (deniseyu) [inline spec/problem_spec.rb]: expect(subject.prime?(2)).to eq true 

is equivalent to this

File: spec/problem_spec.rb [hunk 3]
```diff
@@ -0,0 +1,31 @@
+require 'problem'
+
+describe 'Prime Problem' do
```
Comment (deniseyu) [inline spec/problem_spec.rb]: if you are testing a class and all of the methods within the class, pass the actual class as an argument here. 

i.e. describe Problem do ....

File: spec/problem_spec.rb [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  it 'calculates all prime numbers less than 10,000' do
+    expect(primer.sum(10000)).to eq 5736396
+  end
+
+
+
+
+
+
+end
```
Comment (wardymate) [inline spec/problem_spec.rb]: Great work on a solution. Loads of tests. Possibly refactor.

Erik can you pass back to Nadav

**Pull request closed without merging**